libpod/runtime_migrate_freebsd.go

```diff
@@ -0,0 +1,15 @@
+//go:build !remote
+
+package libpod
+
+import (
+	"errors"
+)
+
+func (r *Runtime) stopPauseProcess() error {
+	return nil
+}
+
+func (r *Runtime) Migrate(newRuntime string) error {
+	return errors.New("not implemented (*Runtime) migrate")
+}
```

Comment:
should this actually be stubbed on freebsd?
This will cause `podman system migrate` to fail with this rather useless error as well,
besides the stopPauseProcess() call I do not see anything linux specific in there and with this change you NOP'ed that call so I would assume it would just work fine